tests/test_qutip.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for qgrad implementation of qutip functions"""
+
+from qgrad.qutip import rot, fidelity
+
+
```

Comment:
Only 1 blank line between each group of imports (Standard/third-party/local)

Reply:
Thanks. I always mix it up. Two blank lines between class def and functions and one between imports.